Source/WebKit/UIProcess/API/glib/WebKitWebContext.cpp

```diff
@@ -1942,6 +1942,29 @@ void webkit_web_context_send_message_to_all_extensions(WebKitWebContext* context
         process->send(Messages::WebProcess::SendMessageToWebProcessExtension(webkitUserMessageGetMessage(message)), 0);
 }
 
+/**
+ * webkit_web_context_get_process_ids:
+ * @context: the #WebKitWebContext
+ * @result: a #GList
+ *
+ * Get the process IDs for the @context.
+ *
+ * Returns: the @result #GList with the process ids appended.
+ *
+ * Since: 2.49.2
+ */
+GList* webkit_web_context_get_process_ids(WebKitWebContext* context, GList* result)
```

Comment:
Why is the list an input parameter?

Anyway, best practice is to [never use GList in APIs](https://www.bassi.io/articles/2023/02/20/bindable-api-2023/). Use a null-terminated array instead. I would make the return type `gint64*`. This works elegantly because if `processID()` returns 0, then applications will see that as the trailing null and stop processing the list, but that's also not obvious, so I would add a comment to mention that.